pkg/kubelet/kubelet.go

```diff
@@ -1976,9 +1976,10 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 }
 
 // dispatchWork starts the asynchronous sync of the pod in a pod worker.
-// If the pod is terminated, dispatchWork
+// If the pod is terminated, dispatchWork will perform no action.
 func (kl *Kubelet) dispatchWork(pod *v1.Pod, syncType kubetypes.SyncPodType, mirrorPod *v1.Pod, start time.Time) {
 	if kl.podIsTerminated(pod) {
```

Comment:
There may be a case that the container's status is running in terminating pod. So podIsTerminated() returns false here and kl.statusManager.TerminatePod(）is never called.